### PR TITLE
Create method for getting object by

### PR DESCRIPTION
**Pull request by letrungkien0210**
Hi @jakubknejzlik ,
Please review.
Thanks.

### CODE REVIEW

File: lib/graphql.schema.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ const getSchemaFromModel = model => {
     const entity = model.entities[entityName];
     const queryName = `get${inflection.pluralize(entity.name)}`;
     queries[queryName] = _getItemsQueryFromEntity(entity);
+
+    const queryById = `get${entity.name}ById`;
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: Please rename to get${entity.name}

File: lib/graphql.schema.js [hunk 2]
```diff
@@ -52,6 +55,21 @@ const _getItemsQueryFromEntity = entity => {
   };
 };
 
+const _getItemQueryByIdFromEntity = (entity) => {
+  return {
+    type: _getObjectTypeFromEntity(entity),
+    args: {
+      id: {
+        description: `Fetch the ${entity.name} by id`,
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: Move this description (of attribute) to query description and here use 'Id of fetched entity ${entity.name}'

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik , I fixed your comments.
Please check.

File: lib/graphql.schema.js [hunk 3]
```diff
@@ -52,6 +55,21 @@ const _getItemsQueryFromEntity = entity => {
   };
 };
 
+const _getItemQueryByIdFromEntity = (entity) => {
+  return {
+    type: _getObjectTypeFromEntity(entity),
+    description: `Id of fetched entity ${entity.name}`,
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: This description should be "Fetch ${entity.name} by id"

File: lib/graphql.schema.js [hunk 4]
```diff
@@ -52,6 +55,21 @@ const _getItemsQueryFromEntity = entity => {
...
+    args: {
+      id: {
+        type: GraphQLInt
```
Comment (jakubknejzlik) [inline lib/graphql.schema.js]: Add description "Id of fetched entity ${entity.name}" here

**Comment by letrungkien0210** [comment]
Hi @jakubknejzlik, I updated the description.
Please check

**Comment by jakubknejzlik** [comment]
👍

**Pull request merged**